src/sdk/src/store_sdk.rs

```diff
@@ -31,31 +33,82 @@ use tonic::Status;
 pub struct StoreSDK {
     client: Arc<StorageNodeClient<tonic::transport::Channel>>,
     signer: Db3Signer,
+    session: SessionManager,
 }
 
 impl StoreSDK {
     pub fn new(
         client: Arc<StorageNodeClient<tonic::transport::Channel>>,
         signer: Db3Signer,
     ) -> Self {
-        Self { client, signer }
+        Self {
+            client,
+            signer,
+            session: SessionManager::new(),
+        }
+    }
+
+    pub async fn restart_session(&mut self) -> std::result::Result<(String, i32), Status> {
+        match self.validate_query_session() {
+            Ok(_) => {
+                let query_session_info = QuerySessionInfo {
+                    id: self.session.get_session_id(),
+                    query_count: self.session.get_session_query_count(),
+                    start_time: self.session.get_start_time(),
+                    status: format!("{:?}", self.session.get_session_status()),
+                };
+                let mut buf = BytesMut::with_capacity(1024 * 8);
+                query_session_info
+                    .encode(&mut buf)
+                    .map_err(|e| Status::internal(format!("{}", e)))?;
+                let buf = buf.freeze();
+                let signature = self
+                    .signer
+                    .sign(buf.as_ref())
+                    .map_err(|e| Status::internal(format!("{}", e)))?;
+                let r = RestartSessionRequest {
+                    query_session_info: buf.as_ref().to_vec(),
+                    signature,
+                };
+                println!("{:?}", query_session_info);
```

Comment:
remove println

Reply:
Resolved in https://github.com/dbpunk-labs/db3/pull/146